src/test/pages/EventDetails.test.tsx

```diff
@@ -104,6 +104,24 @@ describe("EventDetails", () => {
     expect(googleCalendarButton()).not.toBeInTheDocument();
   });
 
+  it.each(["ADMIN", "EVENT_MANAGER", "EVENT_LEADER"] as Role[])(
+    "shows hub name for event if user is %s",
+    async (role) => {
+      await setupTest({ role: role, event: mockEvent });
+      const hub = screen.getByTestId("event-hub");
+      expect(hub).toBeInTheDocument();
+    },
+  );
+
+  it.each(["STUDENT", "TEACHER", "CONTENT_EDITOR", "ANONYMOUS"] as Role[])(
+    "does not show hub name for %s user",
+    async (role) => {
+      await setupTest({ role: role, event: mockEvent });
+      const hub = screen.queryByTestId("event-hub");
+      expect(hub).toBeNull();
```

Comment:
"does not show" tests are risky - are you absolutely certain all hooks have completed before running this? Otherwise you could have a test where it passes...sometimes.

Reply:
Made a small change to make sure event title is appearing before it checks for the lack of hub information.